code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1626,3 +1626,45 @@
 /mob/living/carbon/human/proc/pulse()
 	var/obj/item/organ/internal/heart/H = internal_organs_by_name[BP_HEART]
 	return H ? H.pulse : PULSE_NONE
+
+//Proc for Cultmode to apply corruption to the mob
+//TODO: Add cooldown.
+/mob/living/carbon/human/proc/corrupt(var/amount)

```

Comment:
This should be a variable on the brain imo

Also, make sure amount is an absolute value (so do abs(amount) first)

Might also want to do this in the usual format of setCorruption and adjustCorruption